Common/Charting.cs

```diff
@@ -135,6 +139,17 @@ public class Series
         /// </summary>
         public SeriesType SeriesType = SeriesType.Line;
 
+        /// <summary>
+        /// Color the series 
+        /// </summary>
+        [JsonConverter(typeof(ColorJsonConverter))]
+        public Color Color = Color.Black;
```

Comment:
Please make this nullable and default to null --- if default to black, and the user adds 10 series, they will all be black :)